### PR TITLE
feat(o-shop): Register form component

### PR DESCRIPTION
**Pull request by coffee-intocode**

### CODE REVIEW

**Review by ahaly92 — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { Route, Switch } from 'react-router';
-import { PrivateRoute } from './components/Shared/PrivateRoute';
 
+import './App.scss';
+import { PrivateRoute } from './components/Shared/PrivateRoute';
```
Comment (BarretteBanner) [inline src/App.js]: Since we are only exporting one thing from that file lets not use the {} so we keep things consistent

Comment (BarretteBanner) [inline src/App.js]: Let's also make file names look like shared/private-route
So make it all lowercase and use dashes to separate

File: src/components/Auth/RegisterForm.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  const onsubmit = () => {
+    setSubmitted(true);
+  };
+
+  const formChange = (event) => {
+    const { name, value } = event.target;
+    setInputs({ ...inputs, [name]: value });
+  };
+
+  const isEmpty = (obj) => {
```
Comment (BarretteBanner) [inline src/components/Auth/RegisterForm.js]: let's not abbreviate words

File: src/components/Auth/RegisterForm.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    for (var key in obj) {
+      if (obj[key] === '') return true;
```
Comment (BarretteBanner) [inline src/components/Auth/RegisterForm.js]: Let's practice using an early return here. Also, if possible can we use let instead of var

**Pull request merged**